File: Modul6/C/PRAK605-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main() {
    int n;

    scanf("%d", &n);

    int A[n][n];
    int B[n][n];

    printf("Matriks A\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &A[i][j]);
        }
    }
    printf("Matriks B\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &B[i][j]);
        }
    }

    int AXB[n][n];
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            AXB[i][j] = 0;
            for (int k = 0; k < n; k++) {
                AXB[i][j] += A[i][k] * B[k][j];
            }
        }
    }

    printf("\nMatriks AXB\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("%d ", AXB[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul6/Python/PRAK605-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
n = int(input())

A = []
B = []

print("Matriks A")
for _ in range(n):
    row = list(map(int, input().split()))
    A.append(row)

print("Matriks B")
for _ in range(n):
    row = list(map(int, input().split()))
    B.append(row)

AXB = [[0 for _ in range(n)] for _ in range(n)]

for i in range(n):
    for j in range(n):
        for k in range(n):
            AXB[i][j] += A[i][k] * B[k][j]

print("\nMatriks AXB")
for i in range(n):
    for j in range(n):
        print(AXB[i][j], end=" ")
    print()
